internal/olm/manager.go

```diff
@@ -42,15 +42,15 @@ type Manager struct {
 func (m *Manager) initialize() (err error) {
 	m.once.Do(func() {
 		if m.Client == nil {
-			cfg, err := config.GetConfig()
-			if err != nil {
-				err = errors.Wrapf(err, "failed to get Kubernetes config")
+			cfg, errConf := config.GetConfig()
```

Comment:
Nite: I prefer `cerr` for both cases.

Reply:
Sure, changed as your suggestion.
Thanks.